resources/views/components/user/profile/primary-meta.blade.php

```diff
@@ -24,6 +25,12 @@
 $shouldMoveRoleToNextLine =
     $hasVisibleRole
     && ((mb_strlen($roleLabel) >= 12 && mb_strlen($user->User) >= 12) || mb_strlen($user->User) >= 16);
+
+$previousUsernames = '';
+if ($me && $me->can('viewDisplayNameHistory', $user)) {
+    $previousUsernames = (new BuildDisplayNameHistoryAction())->execute($user);
+}
+$usernameTitle = $previousUsernames ? "Username history:\n{$previousUsernames}" : '';
```

Comment:
If the user doesn't have previous usernames, this still shows the creation line.

![image](https://github.com/user-attachments/assets/310d722d-798c-4142-a648-cda938d938c9)

Reply:
Whoops, addressed in latest.

Additionally, I bumped up the contrast of the underline by 1 notch. On a separate monitor from the one I normally use, I was finding it to be imperceptible.